<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistributionModelingDialog</class>
 <widget class="QDialog" name="DistributionModelingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0" columnminimumwidth="0,2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>1) Set parameters and run:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnParameters">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/setting16</normaloff>:/icons/setting16</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>2) Plot result:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnPlot">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/plot16</normaloff>:/icons/plot16</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>3) Save distribution model:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/save16</normaloff>:/icons/save16</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnDismiss">
        <property name="text">
         <string>Dismiss</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>DistributionModelingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnParameters</sender>
   <signal>clicked()</signal>
   <receiver>DistributionModelingDialog</receiver>
   <slot>onParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlot</sender>
   <signal>clicked()</signal>
   <receiver>DistributionModelingDialog</receiver>
   <slot>onPlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>DistributionModelingDialog</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onParameters()</slot>
  <slot>onSave()</slot>
  <slot>onPlot()</slot>
 </slots>
</ui>
